tests/regression/echotest/errors-warnings-included-expected.echo

```diff
@@ -106,7 +106,7 @@ ECHO: undef
 WARNING: Ignoring unknown module 'unknownIncuse' in file sub2/incuse.scad, line 2
 WARNING: Ignoring unknown variable 'xyz' in file sub1/errors-warnings-incuse.scad, line 2
 ECHO: undef
-WARNING: Ignoring unknown function 'test' in file sub1/errors-warnings-incuse.scad, line 3
+WARNING: Ignoring unknown function 'test' in file errors-warnings-incuse.scad, line 3
```

Comment:
This appears to be a regression as far as I can tell.

Reply:
Nevermind, it was already fixed in https://github.com/openscad/openscad/commit/cc4121ed011321a9ecf5769c00fcf0f6a1372a1e